src/signals_notebook/users/user.py

```diff
@@ -0,0 +1,183 @@
+import cgi
+import logging
+import mimetypes
+from datetime import datetime
+from typing import cast, Optional
+
+from pydantic import BaseModel, Field, PrivateAttr
+
+from signals_notebook.api import SignalsNotebookApi
+from signals_notebook.common_types import File, Response, ResponseData
+
+log = logging.getLogger(__name__)
+
+
+class UserRoleBody(BaseModel):
+    id: str
+    name: str
+
+
+class UserCreationBody(BaseModel):
+    alias: str
+    country: str
+    email: str = Field(alias='emailAddress', allow_mutation=False)
+    first_name: str = Field(alias='firstName')
+    last_name: str = Field(alias='lastName')
+    organization: str = Field(alias='organization')
+    roles: Optional[list[UserRoleBody]]
+
+    class Config:
+        validate_assignment = True
+        allow_population_by_field_name = True
+
+
+class User(BaseModel):
+    is_enabled: Optional[bool] = Field(alias='isEnabled', allow_mutation=False)
+    id: str = Field(alias='userId', allow_mutation=False)
+    username: str = Field(alias='userName', allow_mutation=False)
+    email: str = Field(allow_mutation=False)
+    alias: Optional[str] = Field(alias='alias')
+    first_name: str = Field(alias='firstName')
+    last_name: str = Field(alias='lastName')
+    country: str = Field(alias='country')
+    organization: str = Field(alias='organization')
+    created_at: datetime = Field(alias='createdAt', allow_mutation=False)
+    last_login_at: Optional[datetime] = Field(alias='lastLoginAt', allow_mutation=False)
+    _picture: Optional[File] = PrivateAttr(default=None)
+    _groups = PrivateAttr(default=None)
+    _relationships: dict = PrivateAttr(default=None)
+
+    class Config:
+        validate_assignment = True
+        allow_population_by_field_name = True
+
+    @classmethod
+    def set_relationships(cls, value: dict):
+        cls._relationships = value
+
+    @property
+    def roles(self):
```

Comment:
Should we discuss about lazy load for this class?
Roles or groups? Which of them is more important or this thing?